## Publisher Information
attitude_pub_name: "attitude"
imu_pub_name: "imu"
velocity_pub_name: "velocity"
position_pub_name: "local_position"
height_pub_name: "height_above_takeoff"

## Subscriber Information
pos_ctrl_sub_name: "flight_control_setpoint_ENUposition_yaw"
vel_ctrl_sub_name: "flight_control_setpoint_ENUvelocity_yaw"


## Position Control Parameters
# A: x, y, z, u, v, w, theta, omega
pos_A_x: 1
pos_A_y: 1
pos_A_z: 1
pos_B_u: 1
pos_B_v: 1
pos_B_w: 1

pos_K_pos_coeff: 1
pos_K_vel_coeff: 1.7321
K_yaw: 1


## Velocity Control Parameters
vel_B_u: 1
vel_B_v: 1
vel_B_w: 1
K_yaw_rate: 1.5

## Takeoff and landing  Service parameters
takeoff_land_service_name: 'dji_sim/drone_task_control'
takeoff_altitude:     1
takeoff_vel_max:      1.5
takeoff_pos_error_k:  1

landing_altitude:     0.0 # 0.55
landing_vel_max:      1
landing_pos_error_k:  0.75

## Visual Parameters:
transform_z_offset: 0.00

live_flight: false
